formwidgets/menuitems/assets/js/menu-items-editor.js

```diff
@@ -238,6 +246,11 @@
             else {
                 var $input = $('[name="'+property+'"]', $popupContainer).not('[type=hidden]')
                 setPropertyOnElement($input, val)
+                var RLField = $('[name="RLTranslate['+defaultLocale+']['+property+']"]', self.$popupContainer)
```

Comment:
variable should be `$defaultLocaleField`, jQuery element variables should be prefixed with a `$`.

Reply:
@LukeTowers In this case I'm refering to the data-default-locale attribute... but I'll fix the other places where it should be '$'

Reply:
or you mean replace RLField by "$defaultLocaleField" ?

Reply:
> or you mean replace RLField by "$defaultLocaleField" ?

That's correct